src/js/pages/services/ServicesTab.js

```diff
@@ -53,7 +54,8 @@ var ServicesTab = React.createClass({
 
   getInitialState: function () {
     return Object.assign({}, DEFAULT_FILTER_OPTIONS, {
-      isServiceGroupFormModalShown: false
+      isServiceGroupFormModalShown: false,
+      isDeployServiceModalShown: false
```

Comment:
I'd like more consistency in naming here. Given we already have `ServiceGroupFormModal.js`, I'd suggest naming our new modal `ServiceFormModal.js`. As we'll likely re-use this also for editing a Service, I would not have the `Deploy` in its name.